metrics/grafana/pd.json

```diff
@@ -862,7 +862,6 @@
           "interval": "",
           "intervalFactor": 2,
           "legendFormat": "",
-          "metric": "pd_server_tso",
```

Comment:
Does the grafana's official documentation explain what this field does?

Reply:
Oh, got it https://github.com/tikv/pd/pull/8382#issuecomment-2219828864

Reply:
Can you give me an approve?